### PR TITLE
(feat) dont load the beaker if litmus is there

### PR DESCRIPTION
**Pull request by tphoney**

### CODE REVIEW

File: lib/puppetlabs_spec_helper/rake_tasks.rb [hunk 1]
```diff
@@ -4,11 +4,13 @@
 require 'tmpdir'
 require 'pathname'
 require 'puppetlabs_spec_helper/version'
-require 'puppetlabs_spec_helper/tasks/beaker'
 require 'puppetlabs_spec_helper/tasks/fixtures'
 require 'puppetlabs_spec_helper/tasks/check_symlinks'
 require 'English'
 
+# dont load beaker if litmus is present
+require 'puppetlabs_spec_helper/tasks/beaker' unless Bundler.rubygems.find_name('puppet_litmus').any?
```
Comment (ekohl) [inline lib/puppetlabs_spec_helper/rake_tasks.rb]: This broke: `NameError: uninitialized constant Bundler`

[Rakefile](https://github.com/theforeman/foreman-infra/blob/06ad0cb12d5d4b9d239669b48d20168c984b14f8/puppet/Rakefile) and [Gemfile](https://github.com/theforeman/foreman-infra/blob/06ad0cb12d5d4b9d239669b48d20168c984b14f8/puppet/Gemfile) for reference.

Comment (pmcmaw) [reply lib/puppetlabs_spec_helper/rake_tasks.rb]: Thanks for letting us know @ekohl 
We will work on a fix and have it released as soon as we possibly can.

Comment (ekohl) [reply lib/puppetlabs_spec_helper/rake_tasks.rb]: Thanks! Perhaps it should check if beaker is actually present? In many cases people just exclude beaker unless they're running acceptance tests in their Gemfiles.

Comment (pmcmaw) [reply lib/puppetlabs_spec_helper/rake_tasks.rb]: This issue is being tracked here: https://tickets.puppetlabs.com/browse/MODULES-8778
:-)

**Pull request merged**